configuration.yaml

```diff
@@ -37,7 +37,7 @@ git_services:
     - fedrepo_req
     - module_diff
     - fedora-ci/messages
-    - taskotron/resultsdb
+    # - taskotron/resultsdb
```

Comment:
I'd prefer if you just removed it instead of commenting out. We can always revert the commit if we need the value back.

Reply:
Agree. Added commit to remove all commented out repos, PTAL.